great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -221,14 +230,48 @@ def get_batch_data_and_markers(
 Please check your config."""
                 )
             batch_spec.batch_data = "SparkDataFrame"
+
+        elif isinstance(batch_spec, AzureBatchSpec):
+            reader_method: str = batch_spec.reader_method
+            reader_options: dict = batch_spec.reader_options or {}
+            path: str = batch_spec.path
+            azure_url = AzureUrl(path)
+            try:
+                credential = self._azure_options.get("credential")
+                storage_account_url = azure_url.account_url
+                if credential:
+                    self.spark.conf.set(
+                        "fs.wasb.impl",
+                        "org.apache.hadoop.fs.azure.NativeAzureFileSystem",
+                    )
+                    self.spark.conf.set(
+                        "fs.azure.account.key." + storage_account_url, credential
+                    )
+                reader: DataFrameReader = self.spark.read.options(**reader_options)
+                reader_fn: Callable = self._get_reader_fn(
+                    reader=reader,
+                    reader_method=reader_method,
+                    path=path,
+                )
+                batch_data = reader_fn(path)
+            except AttributeError:
+                raise ExecutionEngineError(
+                    """
+                    Unable to load pyspark. Pyspark is required for SparkDFExecutionEngine.
+                    """
+                )
```

Comment:
Could we write a test where we hit this? Or is it not possible because we mock? Just want to ensure complete test coverage

Reply:
@cdkini Not sure right now.  I would have to "soft-corrupt `PySpark`" to remove the reader method (e.g., `.csv()`.  Perhaps for a separate PR just focused on adding this test, which we have not had thus far?